client/transactions/strings.ts

```diff
@@ -22,3 +22,9 @@ export const displayType = {
 	financing_payout: __( 'Loan dispersement', 'woocommerce-payments' ),
 	financing_paydown: __( 'Loan repayment', 'woocommerce-payments' ),
 };
+
+// Mapping of transaction device type string.
+export const sourceDevice = {
+	// android: __( 'Android', 'woocommerce-payments' ),
```

Comment:
IMO, we should remove the commented-out code and reintroduce it later when appropriate.

Reply:
We should also update the tests and snapshots to remove Android.